File: docs/PayAPP-연동API-oapi-0043/examples/SAMPLE_ANDROID/SAMPLE_ANDROID/payapp_android_sample_v3/OApi/src/main/java/com/udid/oapi/data/PABillPaymentResponseData.java
package com.udid.oapi.data;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

/**
 * Create by 김진원
 */
@Getter
@Setter
@ToString(callSuper = true)
@JsonIgnoreProperties(ignoreUnknown = true)
public class PABillPaymentResponseData extends PAResponseData {

    /**
     * 영수증 url
     */
    String CSTURL;

    /**
     * 결제금액
     */
    String price;

    /**
     * 결제요청번호
     */
    int mul_no;

}

File: docs/PayAPP-연동API-oapi-0043/examples/SAMPLE_IOS/SAMPLE_IOS/payapp_ios_sample_v3/OAPI/model/data/PABillPaymentResponseData.swift
//
//  s2.swift
//  Payapp_iOS_Sample_V2
//
//  Created by 조상현 on 2022/09/26.
//

import ObjectMapper

class PABillPaymentResponseData: PAResponseData {

    
    /**
     * 영수증 url
     */
    var CSTURL:String?

    /**
     * 결제 금액
     */
    var price:String?
    
    /**
     * 결제요청번호
     */
    var mul_no:String?
 
    required convenience init?(map: Map){
        self.init()
    }
    
    override init(){
        super.init()
    }
    
    override func mapping(map: Map) {
        super.mapping(map: map)
        CSTURL                      <- map["CSTURL"]
        price                       <- map["price"]
        mul_no                      <- map["mul_no"]
    }
    
}
